apps/fishing-map/.build.env.sample

```diff
@@ -0,0 +1,16 @@
+BASIC_AUTH_PASS=default
```

Comment:
do we need two env files ? why not reuse the .env.sample we already have ?

Reply:
I decided to use two separate .env files to avoid conflicts because one (`.env`) contains the variables that the app needs to run locally; but the other (`.build.env`) has the variables needed in the build processes (`cloudbuild` and `docker-compose`) to build and deploy the app properly.